### PR TITLE
Point value system with integration test

### PR DESCRIPTION
**Pull request by pratanothayanon97**
- Only admin is able to edit event.
- Only admin can see the event point.
- Fixed indentation for integration and unit tests

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.create(first_name: 'Pear', last_name: 'Ratano', email: 'pear@gmail.com', password: '123', is_admin: 'true')
```
Comment (amaredia3) [inline db/seeds.rb:9]: Why is this needed?

**Review by amaredia3 — APPROVED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (pratanothayanon97) [reply db/seeds.rb:9]: I use this to test for an admin view. Running rails db:seed will populate this to the database.
